lib/management/user.ts

```diff
@@ -834,11 +834,44 @@ const withUser = (sdk: CoreSdk, managementKey?: string) => {
      * @param loginId The login ID of the user
      * @param password The password to set for the user
      */
+    setTemporaryPassword: (loginId: string, password: string): Promise<SdkResponse<never>> =>
+      transformResponse<never>(
+        sdk.httpClient.post(
+          apiPaths.user.setTemporaryPassword,
+          { loginId, password, setActive: false },
```

Comment:
do we need the `setActive` flag? if those are two different routes?
![image](https://github.com/descope/node-sdk/assets/10514677/1c1b8991-b699-4590-b651-a707d2ca0127)

Reply:
Nop you right! its left over I didnt removed. thanks